src/main/java/su/litvak/chromecast/api/v2/ChromeCast.java

```diff
@@ -221,6 +221,36 @@ public final void setVolume(float level) throws IOException {
     }
 
     /**
+     * @param level volume level from 0 to 1 to set
+     *
+     *    ChromeCast does not allow you to jump levels too quickly to avoid blowing speakers
+     *              Setting by increment allows us to easily get the level we want
+     *
+     * @see <a href="https://developers.google.com/cast/docs/design_checklist/sender#sender-control-volume">sender</a>
+     */
+    public final void setVolumeByIncrement(float level) throws IOException {
+        Volume volume = this.getStatus().volume;
```

Comment:
Can the `increment` value be zero? In this case we will fall into infinite loop. I think we need to prevent this by adding some kind of check on `increment` value and maybe throw an `ChromeCastException` when `increment` value is zero.